backend.native/tests/interop/j2objc/ExtendsFoo.java

```diff
@@ -0,0 +1,5 @@
+public class ExtendsFoo extends Foo{
+  public int add3(int firstparam, int secondparam, int thirdparam) {
```

Comment:
I'd do a test with an overloaded method to test that all those bits work right.

Reply:
Done.